### ISSUE TITLE
KuCoin - Websocket errors with liquidity mining strategy

### ISSUE THREAD

**Issue by RC-13** [CoinAlpha/hummingbot#3715]
**Describe the bug**
Opening ticket for tracking:
- When running a liquidity mining strategy using Kucoin as the connector and with multiple markets, the following error keeps coming up 
```python
2021-07-13 12:12:01,384 - 3482 - hummingbot.connector.exchange.kucoin.kucoin_api_order_book_data_source - ERROR - Unexpected exception with WebSocket connection. Retrying after 5 seconds...
Traceback (most recent call last):
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/connector/exchange/kucoin/kucoin_api_order_book_data_source.py", line 447, in _collect_and_decode_messages_loop
    async for raw_msg in kucoin_msg_iterator:
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/connector/exchange/kucoin/kucoin_api_order_book_data_source.py", line 233, in __aiter__
    await self.subscribe(self.stream_type, self.trading_pairs)
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/connector/exchange/kucoin/kucoin_api_order_book_data_source.py", line 150, in subscribe
    await KucoinWSConnectionIterator.update_subscription(self.websocket, stream_type, trading_pairs, True)
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/connector/exchange/kucoin/kucoin_api_order_book_data_source.py", line 138, in update_subscription
    "topic": f"/market/match:{convert_to_exchange_trading_pair(market_str)}",
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/connector/exchange/kucoin/kucoin_utils.py", line 52, in convert_to_exchange_trading_pair
    base, quote = hb_trading_pair.split("-")
ValueError: too many values to unpack (expected 2)
```
The open orders are unaffected however if you leave the bot running for a long duration, there is a high chance that the bot will crash.
- Tested on different accounts using existing and newly generated API keys however the WebSocket error keeps coming up. 
- Test other strategies: no WebSocket error
- There is a scenario that this error may happen when running via source build but no issue when using docker build.

**Steps To Reproduce**

Here are the steps to reproduce the issue (see attachments in the section below):
1. Create a liquidity mining strategy and set KuCoin as the exchange.
2. Add at least three to five or more markets.
3. Start the strategy and observe the log pane

**Screenshots**
![image](https://user-images.githubusercontent.com/57189990/125389961-9fe3a400-e3d4-11eb-8701-7c0c646ea222.png)

**Release version**
0.41.0

**Attachments**
[logs_conf_liquidity_mining_1.log](https://github.com/CoinAlpha/hummingbot/files/6805879/logs_conf_liquidity_mining_1.log)

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**

**Comment by dennisocana** [comment]
@RC-13 the same error does not show when using other strategies like pure market making?

**Comment by RC-13** [comment]
@dennisocana Yes, the error doesn't occur on other strategies.

**Comment by dennisocana** [comment]
Fixed in version 0.41 release
